pyproject.toml

```diff
@@ -32,6 +35,7 @@ exclude = '''
 [tool.pytest.ini_options]
 addopts = """
     -n auto
+    -p no:ape_test"
```

Comment:
should this be in the template? probably yes for plugins

Reply:
Yeah, it seems plugin needs this option or else issues from running the tests in a project, it causes CI build failures :(